### PR TITLE
few minor fixes

### PR DESCRIPTION
**Pull request by AllaMaevskaya**

### CODE REVIEW

File: DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/RecPoints.h [hunk 1]
```diff
@@ -20,6 +20,7 @@
 #include "DataFormatsFT0/ChannelData.h"
 #include "CommonDataFormat/RangeReference.h"
 #include "DataFormatsFT0/Digit.h"
+#include "DataFormatsFT0/ChannelData.h"
```
Comment (mslupeck) [inline DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/RecPoints.h]: This is repetition of line 20

File: Detectors/FIT/FT0/calibration/testWorkflow/FT0CalibCollectorWriterSpec.h [hunk 2]
```diff
@@ -35,7 +35,7 @@ class FT0CalibCollectorWriter : public o2::framework::Task
  public:
   void createAndOpenFileAndTree()
   {
-    TString filename = TString::Format("collFT0_%d.root", mCount);
+    TString filename = TString::Format("collFT0.root", mCount);
```
Comment (mslupeck) [inline Detectors/FIT/FT0/calibration/testWorkflow/FT0CalibCollectorWriterSpec.h]: Is this going to work? I mean without %d in the name, the mCount is meaningless, right?

**Review by mslupeck — DISMISSED** [review]

File: Detectors/FIT/FT0/calibration/testWorkflow/FT0CalibCollectorWriterSpec.h (hunk 2 above)
Comment (AllaMaevskaya) [reply Detectors/FIT/FT0/calibration/testWorkflow/FT0CalibCollectorWriterSpec.h]: just forgot to return

**Review by shahor02 — APPROVED** [review]

**Comment by AllaMaevskaya** [comment]
Can it be merged?

**Pull request merged**